swift/common/middleware/proxy_logging.py

```diff
@@ -234,6 +290,11 @@ def log_request(self, req, status_int, bytes_received, bytes_sent,
         if swift_path.startswith('/v1/'):
             _, acc, cont, obj = split_path(swift_path, 1, 4, True)
 
+        perfdata = req.environ.get('swift.perfdata')
+        if perfdata is not None:
+            req.environ.setdefault('swift.log_info', list()).append(
```

Comment:
We are sharing same log entry with https://github.com/open-io/swift/blob/feature/oio-backend/swift/common/middleware/s3api/controllers/base.py#L108

IMO we should update one of them to avoid collusion (and difficult log parsing)